src/cmd-build-with-buildah

```diff
@@ -57,7 +70,11 @@ done
 
 if [ -z "${VERSION}" ]; then
     # let error out if file does not exist
-    VERSION=$(src/config/versionary)
+    if [ -z "${VERSIONARY}" ]; then
+        VERSION=$(src/config/versionary --dev)
+    else
+        VERSION=$(src/config/versionary)
+    fi
```

Comment:
I wonder if another option here in the future is just drop the versionary switch at all and add a `--prod` or `--pipeline` switch and we'd just add that when we run in the pipeline similar to how we add `--versionary` in the pipeline today.

Reply:
Yeah, absolutely. I mentioned this in the commit message. The `--versionary` switch is awkwardly named now, but trying to keep it "pipeline compatible" to minimize changes that would be needed there. But once we cut over, we could simplify things, yeah.